pkgs/tools/graphics/vulkan-extension-layer/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, cmake, writeText, vulkan-headers, jq }:
+
+stdenv.mkDerivation rec {
+  pname = "vulkan-extension-layer";
+  version = "2020-08-25";
+
+  src = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "Vulkan-ExtensionLayer";
+    rev = "352f8e3e269ff2c650da50ce36313547c2a1fbb6";
+    sha256 = "1503z1zj1xvjpry2h7fpg1frx7lcm54zs3azcgiv5i3ar4wqfkz9";
+  };
+
+  nativeBuildInputs = [ cmake jq ];
+
+  buildInputs = [ vulkan-headers ];
+
+  enableParallelBuilding = true;
```

Comment:
You don't need it, cmake enables parallel building by default, see https://github.com/NixOS/nixpkgs/blob/26803925aa41f3ba67d70facc5c020aa20cf70a8/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L127-L130

Reply:
Fixed, thanks!